Update BUILD_COMMANDLINE.md (#1193)
music-encoding/music-encoding: BUILD_COMMANDLINE.md

>      OpenJDK 64-Bit Server VM Zulu17.32+13-CA (build 17.0.2+8-LTS, mixed mode, sharing)

Reviewer: @martha-thomae Could you update this locally and push to the PR? Unfortunately, the GitHub GUI does not allow to select all three lines for a suggestion.

     ```

>      If the version number indicated is lower than `8.0.0` or if the command returns an empty string, please update or install Java according to an installation instruction matching your operating system (to be found on the internet). For MacOS users, **make sure to install the Java Development Kit (JDK).**

Reviewer 2: Also applies for Ant

>      If the version number indicated is lower than `8.0.0` or if the command returns an empty string, please update or install Java according to an installation instruction matching your operating system (to be found on the internet). For MacOS users, **make sure to install the Java Development Kit (JDK).**

Author: I cannot directly commit this suggestion, @bwbohl, because the file was outdated based on some new commits. But I just added these suggestions in https://github.com/music-encoding/music-encoding/pull/1193/commits/908788912915c3c03deea4bfeca65c9ebabcb7bf.

     Apache Ant™ version 1.10.13 compiled on September 27 2020
     ```

     We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html) or any other applicable installation instruction.

>       * **macOS or Linux:** e.g. the Homebrew Package Manager offers an easy installation methods for both a JDK and Apache Ant™ .

Reviewer 2: maybe the phrase should be harmonized with the windows one, which is more neutral?

>       * **Windows:**

Reviewer: ```suggestion
      * **Windows:** e.g. the [Chocolatey Package Manager](https://chocolatey.org) can be used to install both a JDK and Apache Ant™.
```

>       * **Windows:**

Reviewer: see below

     Apache Ant™ version 1.10.13 compiled on September 27 2020
     ```

>      We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html). **MacOS users are advised to install ant using brew.**

Reviewer 3: ```suggestion
     We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html). **macOS users are advised to install ant using [Homebrew](https://brew.sh/).**
```

>      We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html). **MacOS users are advised to install ant using brew.**

Reviewer 2: If we add the others though, this should be a bit more verbose:

```suggestion
We recommend using the latest stable release of Apache Ant™. If your system has an older version of Apache Ant™ installed you might still give it a try though. If the prompt returns an empty string, please refer to the [Apache Ant™ Installation Instructions](https://ant.apache.org/manual/install.html) or any other applicable installation instruction.

 * **macOS or Linux:** e.g. the Homebrew Package Manager offers an easy installation methods for both a JDK and Apache Ant™ .
 * **Windows:** 
 ```

Author: Same thing here: I cannot directly commit this suggestion, @bwbohl, because the file was outdated based on some new commits. But I just added these suggestions in https://github.com/music-encoding/music-encoding/pull/1193/commits/908788912915c3c03deea4bfeca65c9ebabcb7bf.
